app/services/cloudfront/generate_signed_url_service.rb

```diff
@@ -28,16 +39,18 @@ def generate_url
         end
 
       content_disposition =
-        if MIME::Types[blob.content_type].first&.media_type == 'image'
-          'inline'
+        if SAFE_INLINE_FORMATS.include?(
+             MIME::Types[blob.content_type].first&.content_type
+           )
```

Comment:
There's no need to use `MIME::Types` here any more. It'd been used earlier to safely extract the `media_type`.

```suggestion
        if SAFE_INLINE_FORMATS.include?(blob.content_type)
```